internal/indexer/definitions/beyondhd.yaml

```diff
@@ -70,4 +70,5 @@ irc:
           - torrentId
 
     match:
+      infourl: "/details.php?id={{ .torrentId }}"
```

Comment:
https://beyond-hd.me/details.php?id=000000 gives me 404..

Reply:
Ah yeah, it uses `/torrents/url-encoded-torrent-name.torrentID` so we'll have to encode that to work.